open-api/rest-catalog-open-api.yaml

```diff
@@ -3354,6 +3354,8 @@ components:
             type: integer
         properties:
           type: object
+          additionalProperties:
```

Comment:
fixes `properties` to be properly recognized as a <string, string> map rather than <string, any>